src/main/java/org/sonarsource/auth/github/GitHubSettings.java

```diff
@@ -65,6 +65,7 @@ public String clientId() {
   }
 
   public String clientSecret() {
+    System.out.println("Yes, I know, this is bad.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.auth.github%3Asonar-auth-github-plugin&pullRequest=26&issues=AWICt9Xz1ujmGGnTwCEZ&open=AWICt9Xz1ujmGGnTwCEZ)

[ISSUE_KEY]: AWICt9Xz1ujmGGnTwCEZ